linkedevents/tests/test_remap_events.py

```diff
@@ -0,0 +1,28 @@
+import json
+import os
+import tempfile
+
+import pytest
+from django.core.management import call_command
+
+
+@pytest.mark.parametrize("apply", [True, False])
+@pytest.mark.django_db
+def test_remap_events(place, place2, event, apply):
+    with tempfile.TemporaryDirectory() as d:
+        file_name = os.path.join(d, "test.json")
+        with open(file_name, "w") as f:
+            json.dump({place.id: place2.id}, f)
+
+        # Sanity check
+        assert event.location_id == place.id
+        assert place.id != place2.id
+
+        args = ("--apply",) if apply else ()
+        call_command("remap_events", file_name, *args)
+
+        event.refresh_from_db()
+        if apply:
+            assert event.location_id == place2.id
+        else:
+            assert event.location_id == place.id
```

Comment:
Still not a fan of conditional asserting. The goal of a test should not depend on its parameters. You supply parameters and expect some outcome, not supply parameters and expect some outcome unless x is foo in which case you expect outcome. If this test fails, the short summary tells me absolutely nothing:
```
FAILED linkedevents/tests/test_remap_events.py::test_remap_events[False] - AttributeError: 'Place' object has no attribute 'iddd'
```
(Well, to be fair, they rarely do, but with better test naming, they sometimes at least give an idea what's wrong!)

If this is about avoiding repeated code, please make a fixture out of the file setup.